opendebates/settings.py

```diff
@@ -227,6 +228,7 @@ def __contains__(self, item):
     'jquery',
     'lodash',
     'bootstrap',
+    'bootstrap-select',
```

Comment:
we need to install it because the less compilation imports the less files from the package